After importing virtwl.h I get this error : sommelier.c:3839:17: error: 'VIRTWL_IOCTL_NEW_PIPE_READ' undeclared (first use in this function); did you mean 'VIRTWL_IOCTL_NEW_ALLOC'?

My full make compilation error is below: 

cc -c -o sommelier.o sommelier.c -g -Wall `pkg-config --cflags xcb xcb-composite xcb-xfixes wayland-server wayland-client gbm pixman-1` -I. -D_GNU_SOURCE=1 -DWL_HIDE_DEPRECATED=1 -DXWAYLAND_PATH=\"/usr/bin/Xwayland\"
sommelier.c: In function 'xwl_data_offer_receive':
sommelier.c:3839:17: error: 'VIRTWL_IOCTL_NEW_PIPE_READ' undeclared (first use in this function); did you mean 'VIRTWL_IOCTL_NEW_ALLOC'?
         .type = VIRTWL_IOCTL_NEW_PIPE_READ, .fd = -1, .flags = 0, .size = 0,
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~
                 VIRTWL_IOCTL_NEW_ALLOC
sommelier.c:3839:17: note: each undeclared identifier is reported only once for each function it appears in
sommelier.c: In function 'xwl_send_data':
sommelier.c:5835:17: error: 'VIRTWL_IOCTL_NEW_PIPE_READ' undeclared (first use in this function); did you mean 'VIRTWL_IOCTL_NEW_ALLOC'?
         .type = VIRTWL_IOCTL_NEW_PIPE_READ, .fd = -1, .flags = 0, .size = 0,
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~
                 VIRTWL_IOCTL_NEW_ALLOC
Makefile:45: recipe for target 'sommelier.o' failed
make: *** [sommelier.o] Error 1

do you have an idea of what happened?

--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 # Sommelier + Ubuntu + ChromeOS
 
 ## Prerequisites
-Crouton installed
+Crouton downloaded into `~/Downloads/`
 
 ## Run Crouton
 ```bash
@@ -14,26 +14,32 @@ sudo sh crouton -t cli-extra -r artful
 sudo enter-chroot
 ```
 
-## Setup Environment Variables (might want to add to ~/.bashrc)
+## Setup Environment Variables and alias (might want to add to ~/.bashrc)
 ```bash
-export GDK_BACKEND=wayland 
+export GDK_BACKEND=wayland
 export CLUTTER_BACKEND=wayland
 export XDG_RUNTIME_DIR='/var/run/chrome'
 export WAYLAND_DISPLAY=wayland-0
 export DISPLAY=:0
+alias sommelier="${HOME}/Downloads/sommelier/bin/sommelier -X --x-display=:0"
 ```
 
 ## Install Dependencies
 ```bash
-sudo apt-get install -y pkg-config git make xwayland libwayland-dev libgbm-dev gcc libx11-xcb-dev libsystemd-dev libxcb-composite0-dev libxkbcommon-dev libxrender-dev libxtst-dev libpixman-1-dev
+sudo apt-get install -y pkg-config build-essential git make xwayland libwayland-dev libgbm-dev gcc libx11-xcb-dev libsystemd-dev libxcb-composite0-dev libxkbcommon-dev libxrender-dev libxtst-dev libpixman-1-dev
 ```
 
 ## Get and run Sommelier Service
 ```bash
-git clone https://chromium.googlesource.com/chromiumos/containers/sommelier
+cd ~/Downloads/
+git clone https://chromium.googlesource.com/chromiumos/containers/sommelier.git
 cd sommelier
-make
-./sommelier -X --x-display=:0 --no-exit-with-child /bin/sh -c "./sommelierrc"
+git reset --hard 1382ce084cc40790340d672e8b62ec47733cb860
+sed -i -e 's,\$(PREFIX)/bin,\$(BINDIR),g' -e 's,\$(BINDIR)/Xwayland,`which Xwayland`,g' Makefile
+make PREFIX=~/Downloads/sommelier/bin BINDIR=\$\(PREFIX\) SYSCONFDIR=~/Downloads/sommelier/bin/etc
+make install PREFIX=~/Downloads/sommelier/bin BINDIR=\$\(PREFIX\) SYSCONFDIR=~/Downloads/sommelier/bin/etc
+./bin/sommelier -X --x-display=:0 --no-exit-with-child /bin/sh -c "./bin/etc/sommelierrc" &
+
 ```
 
 ## Done!

